src/cswinrt/strings/WinRT.cs

```diff
@@ -49,18 +56,50 @@ internal static unsafe int CoCreateInstance(ref Guid clsid, IntPtr outer, uint c
         [DllImport("api-ms-win-core-com-l1-1-0.dll")]
         internal static extern unsafe int CoIncrementMTAUsage(IntPtr* cookie);
 
-        [DllImport("kernel32.dll", SetLastError = true)]
-        [return: MarshalAs(UnmanagedType.Bool)]
-        internal static extern bool FreeLibrary(IntPtr moduleHandle);
+        internal static bool FreeLibrary(IntPtr moduleHandle)
+        {
+            int lastError;
+            bool returnValue;
+            int nativeReturnValue;
+            {
+                SetLastError(0);
+                nativeReturnValue = PInvoke(moduleHandle);
+                lastError = GetLastError();
+            }
+
+            // Unmarshal - Convert native data to managed data.
+            returnValue = nativeReturnValue != 0;
+            SetLastError(lastError);
+            return returnValue;
+
+            // Local P/Invoke
+            [DllImportAttribute("kernel32.dll", EntryPoint = "FreeLibrary", ExactSpelling = true)]
+            static extern unsafe int PInvoke(IntPtr nativeModuleHandle);
+        }
+
+        internal static unsafe void* TryGetProcAddress(IntPtr moduleHandle, byte* functionName)
```

Comment:
From my perspective, converting this signature from `sbyte*` to `byte*` should be fine and that looks to be also what the .NET source generator generates.  In addition, we are doing explicit casts from `byte*` to `sbyte*` in all the cases when passing it.  But to confirm, @dongle-the-gadget / @Sergio0694, is there anything I am missing on why `sbyte*` can be useful here with respect to #1346?

Reply:
I'd recommend keeping `sbyte*` (which is also what TerraFX does). `LPCSTR` is defined as `char*`, which isn't signed by default, but most compilers treat it as signed. Therefore, `sbyte*` is "more correct" in this scenario.

Reply:
.NET also uses `sbyte*` in several places, eg. in the `string(sbyte*)` constructor. And `sbyte*` is just more correct given that Windows treats `char` as signed by default (cc. @tannergooding). I mean practically speaking I agree it doesn't make much difference here, but FWIW I'd use `sbyte*` for correctness.